sqla_yaml_fixtures.py

```diff
@@ -1,13 +1,19 @@
 __version__ = (0, 1, 2)
 
-import functools
+try:
+    # Python 3
+    from functools import lru_cache
+except ImportError:
```

Comment:
can you add a `# pragma: no cover` for python 2 specific code. This helps target 100% test coverage.